Client/Application/AppDelegate.swift

```diff
@@ -276,3 +276,88 @@ extension AppDelegate {
         window!.rootViewController = rootViewController
     }
 }
+
+// MARK: - Key Commands
+
+extension AppDelegate {
+    override func buildMenu(with builder: UIMenuBuilder) {
+        super.buildMenu(with: builder)
+
+        guard builder.system == .main else { return }
```

Comment:
Although this needs to be in AppDelegate, would be good to have a separate class to handle this menu building so it doesn't add too much bulk inside AppDelegate